packages/pwa/app/hooks/use-add-to-cart-modal.js

```diff
@@ -139,7 +140,7 @@ export const AddToCartModal = () => {
                                         {!!lineItemPrice &&
                                             intl.formatNumber(lineItemPrice, {
                                                 style: 'currency',
-                                                currency: currency || DEFAULT_CURRENCY
+                                                currency: currency || site.defaultCurrency
```

Comment:
```suggestion
                                                currency: currency || site.l10n.defaultCurrency
```

Reply:
It appears that currency in this hook will always be defined. See the explanation [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/391#discussion_r823171565)